cmd/snap/notes.go

```diff
@@ -105,6 +106,7 @@ func NotesFromLocal(snp *client.Snap) *Notes {
 		InCohort:         snp.CohortKey != "",
 		Health:           health,
 		Held:             snp.Hold != nil && snp.Hold.After(timeNow()),
+		InhibitedRefresh: snp.InhibitedRefresh,
```

Comment:
snapstate "RefreshInhibitedTime" already exists so perhaps `RefreshInhibited` to follow the pattern?
There is however still open question on using boolean VS a time (remainder) in which case this can be ignored.